README.md

```diff
@@ -1,55 +1,19 @@
+| Platform      | Supported          | Notes                                                                                                                                                                                             |
+| ------------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| iOS           | :white_check_mark: | Will be the first and only targeted platform at the beginning                                                                                                                                     |
+| Android       | :white_check_mark: | Will come once iOS version is good enough                                                                                                                                                         |
+| Desktop Web   | :white_check_mark: | Would be nice to have but I'd rather focus on mobile versions                                                                                                                                     |
+| Windows Phone | :x:                | React Native's Documentation is mostly focused on Android and iOS, Windows Phone has been [discontinued](https://www.cnet.com/news/windows-10-mobile-features-hardware-death-sentence-microsoft/) |
 
 ## Development
 
-This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app).
-
-### Git workflow
-
-In order to add a new feature/bug fix, follow these steps:
-
-- Get the latest updates
-
-```
-git fetch origin
-git checkout master
-git pull origin master
-```
-
-- Create your own branch
-
-```
-git checkout -b your-branch-name
-```
-
-- Work, work, work on it by editing the code locally then pushing it to your branch
-- Once the feature/bug fix looks correct, create a Pull Request to merge your commits into `master` from `your-branch-name`
-
-#### Branches
-
-Branches should have a clear name, be short and only use `a-z` and `-`.
-
-#### Commits
-
-:cop: Commit messages should be semantic thanks to this [rigid format](https://seesparkbox.com/foundry/semantic_commit_messages), i.e. they should start with one of these values:
-`['chore:', 'docs:', 'feat:', 'fix:', 'refactor:', 'style:', 'test:']`
-
-The next part should be an action, starting with a verb in present tense.
-Complete example: `docs: update readme`
-
-#### Dependencies
-
-When adding dependencies to this project by running `npm install` please make sure to include `--save-dev` option if applicable (`--save` does not exist anymore as it's done by default according to the [documentation](https://docs.npmjs.com/cli/install#description) now). Make sure to commit `package-lock.json` file as well, [it's expected](https://docs.npmjs.com/files/package-lock.json#description).
```

Comment:
I deleted that part. If I receive a PR around it, I can check it directly. If it becomes a frequent problem, I can add the section back.